selectors.yml

```diff
@@ -1,3 +1,5 @@
+# EVM selectors.
+# File doesn't have a family suffix for backwards compatibility.
```

Comment:
Don't fully understand this comment, this file is not part of the public API of this package?

Reply:
I think for non-Go based applications it is, RMN + some observability afaik @andrevmatos might know more